frontend/src/pages/pipelines/global/modelCustomization/trainingHardwareSection/TrainingHardwareProfileFormSection.tsx

```diff
@@ -75,15 +77,27 @@ const TrainingHardwareProfileFormSection: React.FC<TrainingHardwareProfileFormSe
             <Popover
               bodyContent={
                 <>
+                  <Content component={ContentVariants.p}>
+                    This list includes only hardware profiles that have GPU defined.
+                    {projectScopedHardwareProfiles['1'] &&
+                      projectScopedHardwareProfiles['0'].length > 0 && (
```

Comment:
I think we need to include one more condition, that the project-scoped feature flag should be on to show this bullet point.